testing/features/support/drivers/browserstack.rb

```diff
@@ -33,22 +41,16 @@ def valid_browserstack_drivers
       ]
     end
 
-    def browser_type_class
-      if specialized_driver?
-        Module.const_get("Drivers::Browserstack::#{browser.to_s.pascalize}")
-      else
-        AutomationLogger.debug("Non-specialized driver requested, will use standard Base Driver")
-        Drivers::Browserstack::Base
-      end
-    end
-
-    def specialized_driver?
-      %i[
-        android
-        chrome
-        internet_explorer
-        ios
-      ].include?(browser)
+    def configurable_capabilities
+      {
+        build_name: PayloadValuesGenerator.build_name,
+        project_name: "Public-Website tests",
```

Comment:
Can we rename this so it doesn't reference public website?

Reply:
Sure, thanks for noticing David!